### PR TITLE
Cumulative Fixes:

### PR DESCRIPTION
**Pull request by oWahyudi**
**SCRUM-98**
1. Rewrite edge-connect-pallet benchmarking to align with the new logic
3. Rewrite task-management-pallet benchmarking to align with the new logic
5. Fix  Mock and Unit Test file
7. Rewrite status-aggregator-pallet benchmarking to align with the new logic

**SCRUM-99**
1. add batch script to generate benchmark file run_benchmarks.sh
2. generate pallet-timestamp weight file and configure WeightInfo runtime
3. generate pallet-sudo weight file and configure WeightInfo runtime
4. generate cumulus-pallet-xcmp-queue weight file and configure WeightInfo runtime
5. generate pallet-message-queue weight file and configure WeightInfo runtime
6. generate pallet-session weight file and configure WeightInfo runtime
7. generate pallet-collator-selection weight file and configure WeightInfo runtime
8. generate pallet-balances weight file and nothing to configure.
9. adjust frame-weight-template to include pallet prefix

**SCRUM-101**

1. Set `FeeManager` to `XcmFeeToAccount` to handle how fees are routed when executing XCM messages across different chains or consensus system.

**SCRUM-100**

1. Implement an exponential fee mechanism to charge adequate message delivery

### CODE REVIEW

**Comment by oWahyudi** [comment]
**SCUM-100**

1. Implement an exponential fee mechanism to charge adequate message delivery

File: runtime/src/apis.rs [hunk 1]
```diff
@@ -248,7 +248,7 @@ impl_runtime_apis! {
 			use frame_benchmarking::{BenchmarkError, Benchmarking, BenchmarkBatch};
 			use super::*;
 
-			use frame_system_benchmarking::Pallet as SystemBench;
+			//use frame_system_benchmarking::Pallet as SystemBench;
```
Comment (ZCalz) [inline runtime/src/apis.rs:251]: Was this removed for testing?

**Review by ZCalz — APPROVED** [review]

File: runtime/src/apis.rs (hunk 1 above)
Comment (oWahyudi) [reply runtime/src/apis.rs:251]: Noted, will check on it

Comment (oWahyudi) [reply runtime/src/apis.rs:251]: It was declared but never used.  I will restore it back

**Pull request merged**